__mocks__/@actions/github/index.ts

```diff
@@ -10,4 +10,10 @@ export const clearContextMock = () => {
     });
 };
 
-export const getOctokit = jest.fn();
+export const getOctokit = jest.fn().mockReturnValue({
```

Comment:
Calling `mockReturnValue` here will mock the value for all tests, which can be confusing for other people. Instead, you can call it directly in your test. For example:

```ts
import { getOctokit } from '@actions/github';

(getOctokit as jest.Mock<any, any>).mockReturnValue({ /* your value */ });

```

Reply:
Done. :)